client/src/components/JobList.vue

```diff
@@ -75,13 +61,15 @@
         chosenJob: null,
         availabilityDate: moment(),
         country: 'anyCountry',
+        status: 'anyStatus',
       };
     },
 
     computed: {
       displayedJobs() {
         const countryJobs = countryFilter.filter(this.jobsFromApi, this.country);
-        return jobsSorter.sort(countryJobs, this.availabilityDate);
+        const statusJobs = statusFilter.filter(countryJobs, this.status);
```

Comment:
Peut être mieux un let jobsToDisplay pour remplacer les deux const ?
Ou alors ce n'est pas possible de faire un map pour chainer les deux filtres ?

Reply:
Peut être mieux un let jobsToDisplay pour remplacer les deux const ?
=> Je n'aime pas jobsToDisplay (pour countryJobs), parce qu'on n'affiche pas tous les jobs de tous les statuts. 
Pourquoi pas pour "statusJob"... mais j'ai voulu garder une cohérence.

Je propose filteredJobs.

Reply:
En fait, j'aime pas : 
displayedJobs() {
        let filteredJobs = countryFilter.filter(this.jobsFromApi, this.country);
        filteredJobs = statusFilter.filter(filteredJobs, this.status);
        return jobsSorter.sort(filteredJobs, this.availabilityDate);
      },

=> le filter sur un filteredJobs, je n'aime pas. Je ne trouve pas ça cohérent. 
Bref je laisse

Reply:
Ou alors ce n'est pas possible de faire un map pour chainer les deux filtres ?
=> Non.
Map sert à appliquer une opération sur chaque élément de la liste, pris un par un.
Ici on envoie toute la liste à des fonctions.